src/main/kotlin/com/openlattice/organizations/processors/UpdateOrganizationExternalDatabaseTable.kt

```diff
@@ -0,0 +1,48 @@
+package com.openlattice.organizations.processors
+
+import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
+import com.openlattice.edm.requests.MetadataUpdate
+import com.openlattice.organization.OrganizationExternalDatabaseTable
+import java.util.*
+
+class UpdateOrganizationExternalDatabaseTable(val update: MetadataUpdate) :
+        AbstractRhizomeEntryProcessor<UUID, OrganizationExternalDatabaseTable, OrganizationExternalDatabaseTable?>() {
+
+    override fun process(entry: MutableMap.MutableEntry<UUID, OrganizationExternalDatabaseTable>): OrganizationExternalDatabaseTable? {
+        val table = entry.value
+
+        if (update.title.isPresent) {
```

Comment:
Minor, but I think you can use ifPresent, which is just a little cleaner since you can avoid the .get() call 

i.e.
```
update.title.ifPresent( title -> table.title = title )
```
(or something like that, not 100% on the syntax)

Reply:
nice will do